index.bs

```diff
@@ -2072,6 +2072,8 @@ When registering a new credential, represented by a {{AuthenticatorAttestationRe
 
 15. If verification of the attestation statement failed, the [=[RP]=] MUST fail the registration ceremony.
 
+16. Store the public key and the signature counter included in the attestation statement for this authenticator.
```

Comment:
Perhaps we should re-arrange the steps here to be clearer. Step 15 seems to be in direct contradiction with step 14. 

Suggestions: 
13. If the attestation statement |attStmt| verified successfully and is found to be trustworthy, then register the new
    credential with the account that was denoted in the
    {{PublicKeyCredential/[[Create]](options)/options}}.{{MakePublicKeyCredentialOptions/user}} passed to
    {{CredentialsContainer/create()}}, by associating it with the credential ID, credential public key, **and signature counter** contained in
    |authData|'s [=attestation data=], as appropriate for the [=[RP]=]'s systems.

14. If the attestation statement |attStmt| successfully verified but is not trustworthy per step 12 above, the [=[RP]=] SHOULD fail
    the registration ceremony.

    NOTE: However, if permitted by policy, the [=[RP]=] MAY register the credential ID, the credential public key, **and signature counter** but treat the
        credential as one with [=self attestation=] (see [[#sctn-attestation-types]]). If doing so, the [=[RP]=] is asserting there
        is no cryptographic proof that the [=public key credential=] has been generated by a particular [=authenticator=] model.
        See [[FIDOSecRef]] and [[UAFProtocol]] for a more detailed discussion.

Reply:
Done.